src/renderer/modals/Receive/steps/StepAccount.js

```diff
@@ -116,6 +122,32 @@ export default function StepAccount({
           onChangeToken={onChangeToken}
         />
       ) : null}
+      {account && !receiveTokenMode && tokenTypes.length && account && account.type === "Account" && (
```

Comment:
you duplicated account condition + 3 /4 conditions are the same between the two token tips. maybe do something like 

```` javascript

{ account && !receiveTokenMode && tokenTypes.length ? 
        <TokenTips
            textKey={account.type === "TokenAccount" 
                            ? "receive.steps.chooseAccount.verifyTokenType"
                            :  "receive.steps.chooseAccount.warningTokenType"}
            textData={account.type === "TokenAccount" 
              ? {
                  token: account.token.name,
                  tokenType: tokenTypes.map(tt => tt.toUpperCase()).join("/"),
                  currency: mainAccount && mainAccount.currency.name,
                } 
              : {
                  ticker: account.currency.ticker,
                  tokenType: tokenTypes.map(tt => tt.toUpperCase()).join("/"),
                 currency: account.currency.name,
              }}
            learnMoreLink={supportLinkByTokenType[tokenTypes[0]]}
          /> 
    : null }

Reply:
ok